lib/src/sql/statements/info.rs

```diff
@@ -64,6 +64,17 @@ impl InfoStatement {
 					tmp.insert(v.name.to_string(), v.to_string().into());
 				}
 				res.insert("users".to_owned(), tmp.into());
+
+				// Process cluster membership
+				let mut memberships: Vec<Value> = vec![];
+				for cluster_member in run.scan_nd(0).await? {
+					let mut tmp = Object::default();
+					tmp.insert("nodeId".to_string(), cluster_member.name.into());
```

Comment:
Actually, the types we use are `Value::Array` and `Value::Object`. So you can have a `Value::Array` of `Value::Object` and the object can be `String` to `Value::Strand(String)` for example.

Reply:
So, something like this ?

```rust
tmp.insert("nodeId".to_string(), Value::Strand(cluster_member.name.into()));
```